### PR TITLE
SAK-43023 Search enable Assignments

### PR DESCRIPTION
**Pull request by adrianfish**
https://jira.sakaiproject.org/browse/SAK-43023

### CODE REVIEW

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentContentProducer.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/sakaiproject/sakai/pullRequest?prid=4846878&bid=16086844)

File: assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentContentProducer.java [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+    private SearchService searchService;
+    private SiteService siteService;
+    private TransactionTemplate transactionTemplate;
+
+    private List<String> addingEvents = new ArrayList<>();
+    private List<String> removingEvents = new ArrayList<>();
+
+    public void init() {
+
+		if ("true".equals(serverConfigurationService.getString("search.enable", "false"))) {
```
Comment (ottenhoff) [inline assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentContentProducer.java]: tabbing

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentContentProducer.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/sakaiproject/sakai/pullRequest?prid=4846878&bid=16086844)

File: assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentContentProducer.java (hunk 1 above)
Comment (adrianfish) [reply assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentContentProducer.java]: sorted

**Pull request merged**